kfs-ec/src/main/resources/edu/arizona/kfs/module/ec/spring-ec.xml

```diff
@@ -16,5 +16,9 @@
         		<value>classpath:edu/arizona/kfs/module/ec/document/datadictionary/*.xml</value>
         	</list>
         </property>
+        <property name="triggerNames">
```

Comment:
There aren't actually any triggers defined in spring-ec, so this is an unneeded override.